### PR TITLE
fix(install/mesos) changed Mesos reference to DCOS

### PR DESCRIPTION
**Pull request by shashiranjan84**

### CODE REVIEW

File: app/install/dcos.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+links:
+  mesos-aws: "https://dcos.io/docs/1.8/administration/installing/cloud/aws/"
+  mesos-cli: "https://docs.mesosphere.com/1.8/usage/cli/"
+  mesos-gui: "https://docs.mesosphere.com/1.8/usage/webinterface/"
+  marathon: "https://mesosphere.github.io/marathon/"
+  marthon-lb: "https://docs.mesosphere.com/1.7/usage/service-discovery/marathon-lb/usage/"
+  dcos: "https://docs.mesosphere.com/1.8/overview/"
+---
+
+Kong can easily be provisioned on a DC/OS cluster using following
```
Comment (theetrain) [inline app/install/dcos.md]: > ...can `be` provisioned on a DC/OS cluster using `the` following

I have a personal bias against the words 'easily', 'simply', and 'just'.

**Comment by coopr** [comment]
LGTM, @sonicaghi I think you wanted to look at this too

File: app/install/dcos.md (hunk 1 above)
Comment (coopr) [inline app/install/dcos.md]: I thought I saw you change this to "...on a Mesosphere DC/OS cluster..." ?

File: app/install/dcos.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    Using the `marathon-lb-internal.json` file included in the kong-dist-mesos
+    repo, deploy Marathon-lb for internal and external service discovery
+    support:
+
+    ```bash
+    $ dcos package install marathon-lb
+    $ dcos package install --options=marathon-lb-internal.json marathon-lb
+    ```
+
+4. **Deploy a Kong supported database**
```
Comment (theetrain) [inline app/install/dcos.md]: > Deploy a `Kong-supported` database

File: app/install/dcos.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    repo to deploy a PostgreSQL instance in the cluster:
+
+    ```bash
+    $ dcos marathon app add postgres.json
+    ```
+
+5. **Deploy Kong**
+
+    Using the `kong_<postgres|cassandra>.json` file from the
+    kong-dist-mesos repo, deploy Kong in the cluster:
```
Comment (theetrain) [inline app/install/dcos.md]: You may want to link `kong-dist-mesos` to its repo.

Comment (shashiranjan84) [inline app/install/dcos.md]: I don't think its required as we already requesting user to download the repo to the host they working on.

File: app/install/dcos.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  mesos-gui: "https://docs.mesosphere.com/1.8/usage/webinterface/"
+  marathon: "https://mesosphere.github.io/marathon/"
+  marthon-lb: "https://docs.mesosphere.com/1.7/usage/service-discovery/marathon-lb/usage/"
+  dcos: "https://docs.mesosphere.com/1.8/overview/"
+---
+
+Kong can be provisioned on a Mesosphere DC/OS cluster using following
+steps:
+
+The following steps use AWS for provisioning the DC/OS cluster and assumes you
```
Comment (Tieske) [inline app/install/dcos.md]: assume

Comment (shashiranjan84) [inline app/install/dcos.md]: done.

File: app/install/mesos.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ links:
 Kong can easily be provisioned on a Mesosphere DC/OS cluster using following
 steps:
 
-The following steps use AWS for provisioning the DC/OS cluster and assumes you 
+The following steps use AWS for provisioning the DC/OS cluster and assume you
```
Comment (thibaultcha) [inline app/install/mesos.md]: Why leaving this file if we are replacing it?

Comment (shashiranjan84) [inline app/install/mesos.md]: My mistake, removed it.

**Pull request merged**